spec/unit/sources_manager_spec.rb

```diff
@@ -137,22 +148,17 @@ module Pod
 
       describe 'detect cdn repo' do
         it 'cdn master spec repo' do
-          expected = Typhoeus::Response.new(:code => 200, :body => CDN_REPO_RESPONSE)
-          Typhoeus.stub('https://cdn.cocoapods.org/CocoaPods-version.yml', :netrc_file => Netrc.default_path, :netrc => :optional).and_return(expected)
           @sources_manager.cdn_url?('https://cdn.cocoapods.org').should == true
         end
 
         it 'cdn master spec repo by http' do
-          expected = Typhoeus::Response.new(:code => 200, :body => CDN_REPO_RESPONSE)
-          Typhoeus.stub('http://cdn.cocoapods.org/CocoaPods-version.yml', :netrc_file => Netrc.default_path, :netrc => :optional).and_return(expected)
           @sources_manager.cdn_url?('http://cdn.cocoapods.org').should == true
         end
 
         it 'git master spec repo' do
-          expected = Typhoeus::Response.new(:code => 404, :body => 'Not Found')
-          Typhoeus.stub('https://github.com/cocoapods/specs.git/CocoaPods-version.yml', :netrc_file => Netrc.default_path, :netrc => :optional).and_return(expected)
+          stub_as_404('https://github.com/cocoapods/specs.git')
```

Comment:
this used to stub `https://github.com/cocoapods/specs.git/CocoaPods-version.yml` but now its not?

Reply:
it is, `CocoaPods-version.yml` is appended inside that function